5._Governance.md

```diff
@@ -6,57 +6,55 @@ As a community effort organized within the [Supply Chain Integrity WG](https://g
 
 The SLSA Project described in this document is the "Working Group" for purposes of the [Community Specification License 1.0](./1._Community_Specification_License-v1.md).
 
+The SLSA Project is comprised of multiple "Work Streams." These Work Streams represent different tracks of specification work or tools implementation. Work Streams are not strictly tied to unique repositories in the slsa-framework GitHub organization. 
+
 ## 1. Roles.
 
 The Project includes the following roles. Additional roles may be adopted and documented by the Steering Committee.
 
-**1.1. Participants.** "Participants" are those that have (a) made Contributions to the Project subject to the Community Specification License; (b) contributed code or documentation to Project assets other than specifications; and/or (c) otherwise contributed to or participated in the SLSA community, such as by attending meetings.
+**1.1. Participants.** "Participants" are those that have contributed to or participated in the SLSA community, such as by attending meetings, engaging in conversations on Slack, or commenting on documents.
```

Comment:
What's the motivation for removing the 'Community Specification License'?

Reply:
We thought the current governance, where the Participant requirement could be met with any of these three, was overly precise. (a) or (b) meet the requirements for Contributor, and of course a Contributor is a Participant as well.

Reply:
The Community Specification License defined Contributor focused only on work for a specification. For SLSA we wanted something a bit broader that allowed for tooling development work to count as well. For reference here's the referenced definition that we replaced with our more explicit and inclusive text.

9.4. Contributor. “Contributor” means any person or entity that has indicated its acceptance of the License 1) by making a Contribution to the Specification, or 2) by entering into the Community Specification Contributor License Agreement for the Specification. Contributor includes its Affiliates, assigns, agents, and successors in interest.